packages/react/src/contexts/DefaultOptionsContexts.ts

```diff
@@ -2,8 +2,9 @@ import { createContext } from 'react'
 import type { DelayProps, SuspenseProps } from '..'
 import type { OmitKeyOf } from '../utility-types'
 
-export const DelayDefaultPropsContext = createContext<OmitKeyOf<DelayProps, 'children'>>({
+export const DelayDefaultPropsContext = createContext<DelayProps>({
```

Comment:
> In my opinion, if this is named for meaning props of Delay, because Delay have children prop, we should use PropsWithChildren in DelayProps internally
> 
> We can check ErrorBoundaryProps contain type PropsWithChildren internally
https://github.com/bvaughn/react-error-boundary/blob/master/src/types.ts#L18C33-L26

Because of this reason, we had to omit children of each components(Delay, Suspense) intentionally. because we didn't want library users to use default prop for children prop. children prop should be set by use case each time

Reply:
@manudeli Aha I understand the intent, thanks for the quick feedback  🙏